.github/workflows/documentation-validation.yml

```diff
@@ -0,0 +1,20 @@
+name: Markdown Lint
+
+on: [push]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
Change to v3 to avoid 

<img width="953" alt="Pasted Graphic 32" src="https://github.com/user-attachments/assets/c6a11eb3-f049-4b0e-a97c-b0d3e2243129">